### PR TITLE
Issue #277: remove scala.util.parsing.json dependencies (deprecated)

### PR DESCRIPTION
**Pull request by adrapereira**
Removed scala.util.parsing.json dependencies and replaced the json parsing with jackson-module-scala

### CODE REVIEW

File: live/src/main/java/org/dbpedia/extraction/live/storage/JSONCache.scala [hunk 1]
```diff
@@ -119,8 +122,9 @@ class JSONCache(pageID: Long, pageTitle: String) {
       cacheExists = true
       if (cacheObj.json.equals("")) return
 
-      val json: Option[Any] = JSON.parseFull(cacheObj.json)
-      val map: Map[String, Any] = json.get.asInstanceOf[Map[String, Any]]
+      val mapper = new ObjectMapper() with ScalaObjectMapper
+      mapper.registerModule(DefaultScalaModule)
```
Comment (jimkont) [inline live/src/main/java/org/dbpedia/extraction/live/storage/JSONCache.scala]: Not very familiar with this library but should we call `registerModule` on every instantiation?

**Comment by adrapereira** [comment]
The documentation of the library doesn't address this. I now define the mapper in the Object JSONCache and only call registerModule once in initCache. I don't know if that's better...

File: live/src/main/java/org/dbpedia/extraction/live/storage/JSONCache.scala [hunk 2]
```diff
@@ -119,8 +122,9 @@ class JSONCache(pageID: Long, pageTitle: String) {
       cacheExists = true
       if (cacheObj.json.equals("")) return
 
-      val json: Option[Any] = JSON.parseFull(cacheObj.json)
-      val map: Map[String, Any] = json.get.asInstanceOf[Map[String, Any]]
+
+      JSONCache.mapper.registerModule(DefaultScalaModule)
```
Comment (jimkont) [inline live/src/main/java/org/dbpedia/extraction/live/storage/JSONCache.scala]: then I think you should move this one as well

**Comment by adrapereira** [comment]
Do feel it's better this way?

**Comment by jimkont** [comment]
I think so, did you test it locally or just compile?

**Comment by adrapereira** [comment]
I tested it in a local wikipedia NL copy. It appears to be working well. The messages show the number of produced triples for each page processed.

**Comment by jimkont** [comment]
Thanks @adrapereira ! Looks good on a first look. What I need to checked is how `JSONCache.mapper.readValue` behaves on concurrent use. After that we'll merge

**Comment by jimkont** [comment]
ok, this is thread safe [1]. Thanks a lot Andre! #277

[1]  http://wiki.fasterxml.com/ObjectMapper

**Comment by jimkont** [comment]
@adrapereira  the code produces the following error
```
Argument #0 of constructor [constructor for java.util.LinkedHashMap, annotations: [null]] has no property name annotation; must have name when multiple-parameter constructor annotated as Creator
```
could you take a look?

**Comment by adrapereira** [comment]
I'll see if I can reproduce this error. Was there no more information? Line number, file, etc.?

**Comment by jimkont** [comment]
Unfortunately no
From the log it looked like all lookups failed

**Pull request merged**